next-cloudinary/package.json

```diff
@@ -1,21 +1,24 @@
 {
   "name": "next-cloudinary",
   "version": "2.2.0",
-  "main": "dist/next-cloudinary.umd.js",
-  "module": "dist/next-cloudinary.module.js",
-  "umd:main": "dist/next-cloudinary.umd.js",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
```

Comment:
i pulled these definitions from an example that I went off of for https://github.com/colbyfayock/cloudinary-util from the turborepo Examples directory, figure they know what they're doing more than i do :)